backend/prismaCalls.ts

```diff
@@ -42,4 +42,19 @@ export async function getPackageHistories(queryName: apiSchema.PackageName){
         console.log(`Error in getPackageHistories: ${error}`);
         return null;
     }
+}
+
+export async function uploadMetadataToDatabase(metadata: apiSchema.PackageMetadata): Promise<void> {
```

Comment:
Need to upload the response type and also, you need to make a packageHistoryEntry with the associated action that happened (this is for getPackageHistory)